docker-compose.yml

```diff
@@ -9,13 +9,14 @@ services:
     environment:
       - NODE_ENV=dev
       - REDIS_URL=redis://redis
+      - GITHUB_TOKEN
```

Comment:
Why do we need this? `GITHUB_TOKEN` is used only inside the script `init-npm.sh` that is never executed during build phase but only inside circle-ci pipeline.

Reply:
that's because inside the Dockerfile we launch 'yarn install' which would fail in case there's no token (isn't it ?)

Reply:
yarn install will use `.npmrc` that is already copied into the image with the line `COPY /.npmrc /usr/src/app/.npmrc`

Reply:
that states if you have hardcoded the token into the npmrc file that gets copied. another option (which is why we pass the token) is to get it from the environment inside npmrc (the real issue here is that this is not documented into the readme)

Reply:
agree

Reply:
I try to run docker-compose from scratch and during the yarn install phase I've got an error because it cannot connect to repo while downloading @pagopa/italia-ts-commons.
I've added this variable to keep both 2 solutions to inject GITHUB_TOKEN into docker image:
1) by copying npmrc
2) by injecting variables at command line
I agree that if we hardcode GITHUB_TOKEN inside npmrc, we doesn't need to inject the same variable at command line.